spec/api-menu-item-spec.ts

```diff
@@ -179,23 +183,23 @@ describe('MenuItems', () => {
       const win = new BrowserWindow({ show: false, width: 200, height: 200 });
       const item = new MenuItem({ role: 'asdfghjkl' as any });
 
-      const canExecute = execute(item.role, win, win.webContents);
+      const canExecute = execute(item.role as any, win, win.webContents);
```

Comment:
can "any" be avoided?

Reply:
in this case unfortunately not, there are two lists of roles (one documented with camelCase and another one lowercased)